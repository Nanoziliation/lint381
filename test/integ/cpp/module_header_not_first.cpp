// Flag including other headers before the module header.
#include "foo.h"
#include "module_header_not_first.h"
