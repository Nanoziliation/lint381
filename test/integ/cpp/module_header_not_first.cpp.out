module_header_not_first.cpp:3:10: error: Header 'module_header_not_first.h' should be first include in 'module_header_not_first.cpp'
#include "module_header_not_first.h"
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
